### PR TITLE
cmd/tailscaled, wgengine/netstack: always wire up netstack

### PR DESCRIPTION
**Pull request by bradfitz**
Even if not in use. We plan to use it for more stuff later.

### CODE REVIEW

File: wgengine/netstack/netstack.go [hunk 1]
```diff
@@ -54,13 +54,20 @@ type Impl struct {
...
-	e           wgengine.Engine
-	mc          *magicsock.Conn
-	logf        logger.Logf
-	onlySubnets bool // whether we only want to handle subnet relaying
+	// ProcessAll is whether netstack should handle all incoming traffic.
+	ProcessAll bool
+
+	// ProcessSubnets is whether netstack should traffic to
+	// non-local IPs (e.g. it should be a subnet router). This
+	// option is ignored ProcessAll is set anyway.
```
Comment (DentonGentry) [inline wgengine/netstack/netstack.go:62]: "This option is ignored ProcessAll is set anyway."

Something got deleted from that sentence.

**Review by crawshaw — APPROVED** [review]
The code looks fine but I'd like to know if it has any impact on throughput. Any benchmarks we can run?

**Comment by bradfitz** [comment]
> The code looks fine but I'd like to know if it has any impact on throughput. Any benchmarks we can run?

It should be negligible. A method call that reads a false field and does nothing. Nothing actually goes through netstack when it's off.

File: wgengine/netstack/netstack.go (hunk 1 above)
Comment (bradfitz) [reply wgengine/netstack/netstack.go:62]: Whoops. I'll rewrite. And might rename these fields. They're subtle and confusing.

Comment (bradfitz) [reply wgengine/netstack/netstack.go]: PTAL; renamed & rewritten.

**Pull request merged**